### PR TITLE
Hymns for the Miracles of Ewostatewos

### PR DESCRIPTION
**Pull request by smaugustine**
From the [relevant manuscripts](https://github.com/BetaMasaheft/Documentation/issues/1518):
- BL Or 702 is updated by this PR
- Vat Borg et 9 is updated by this PR
- BL Or 829* needs a new record (?) corresponding to Wright CXCIX, it's not totally clear to me how the records for that are being handled
- EMIP 480 needs to be updated in [a different PR](https://github.com/BetaMasaheft/Manuscripts/pull/805)
- EMIP 764 is currently being marked up by @Ralph-Lee-UK and I don't want to cause a conflict with his edits

### CODE REVIEW

**Comment by eu-genia** [comment]
on 829: https://github.com/BetaMasaheft/Documentation/issues/974
I believe that the majority was voting for one record for all the 829* fragments (which would mean no new record but a revision of the existing one(s)) but there was no final decision in the issue

File: VaticanBAV/borget/BAVborget9.xml [hunk 1]
```diff
@@ -61,79 +61,29 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
-                        <incipit xml:lang="gez">ካልዕ፡ ተአምሪከ፡ ኤዎስጣቴዎስ፡ ድንግል፡ አመ፡ ውስተ፡ ባሕር፡ ተሰጥመ፨ እንዘ፡ ይጐይይ፡ እምቀትል፤</incipit>
+                        <locus from="54v" to="58v" facs="0110"/>
+                        <title ref="LIT3053RepCh333"/>
+                        <incipit xml:lang="gez">እምብዙኅ፡ ተአምሪከ፡ ዕዜኑ፡ አሐተ፡ ኤዎስጣቴዎስ፡ ዘትገብር፨ ዓበይተ፡ ኃይላተ፡ መኰንን፡ ዝኁር፨ ሶበ፡ ፈቀደ፡ ነሢተ፡ መቃብሪከ፡ ክብርተ፨</incipit>
                       </msItem>
 
                       <msItem xml:id="ms_i4.4">
-                        <locus target="#55r"/>
-                        <title>Third Miracle</title>
-                        <incipit xml:lang="gez">ሣልስ፡ ተአምሪከ፡ ዝውእቱ፤ ከመ፡ አድኃንኮሙ፡ አባ፡ ለኃብተ፡ እግዚእ፡ ወለወለቱ፨ እመቅሠፍተ፡ ሥራይ፡ ኅሱም፡</incipit>
```
Comment (eu-genia) [inline VaticanBAV/borget/BAVborget9.xml:88]: the incipits of the stanzas, now suppressed, should go to LIT3053RepCh333 I assume?

**Review by eu-genia — APPROVED** [review]

File: VaticanBAV/borget/BAVborget9.xml (hunk 1 above)
Comment (smaugustine) [reply VaticanBAV/borget/BAVborget9.xml:88]: Yep! See https://github.com/BetaMasaheft/Works/blob/8db37e0225f3f06cd7e92ff743a91027c14d6009/3001-4000/LIT3053RepCh333.xml#L64

**Comment by Ralph-Lee-UK** [comment]
@smaugustine just to let you know that at the moment I am not editing EMIP 764, but am just working on detailing the contents of the Cheleqot Selassie collection of manuscripts so that another of our team can do the edits, so please go ahead with editing EMIP 764.

**Pull request merged**